library/general/src/modules/Directory.rb

```diff
@@ -143,6 +147,7 @@ def find_data_file(relative_path)
     publish variable: :moduledir, type: "string"
     publish variable: :scrconfdir, type: "string"
     publish variable: :desktopdir, type: "string"
+    publish variable: :icondir, type: "string"
```

Comment:
I'd add a comment like 
```ruby
# @deprecated Use just a plain icon name without suffix,
#    the UI will automatically find the best suitable icon.
```

(Is that actually correct?)

Reply:
In most widgets, yes, not in RichText yet though